buster/llm_utils/embeddings.py

```diff
@@ -11,6 +12,7 @@
 client = OpenAI()
 
 
+@lru_cache
```

Comment:
the decorator should be removed from the retriever too then

Reply:
the retriever uses this exact same function since last refactor so it shouldnt be necessary to remove anymore